cli/src/commands/jobs.rs

```diff
@@ -228,28 +231,15 @@ impl JobsProject {
     /// Assumes that the clap `matches` has a `project` and `group` arguments
     /// option.
     async fn new(api: &mut PhylumApi, matches: &clap::ArgMatches) -> Result<JobsProject> {
-        let cli_lockfile_type = matches.try_get_one::<String>("lockfile-type").ok().flatten();
-        let cli_lockfile = matches.try_get_one::<String>("LOCKFILE").ok().flatten();
```

Comment:
The new code uses `get_one` instead of `try_get_one`.

I believe the reason for `try_get_one` was to handle the `phylum batch` command (which doesn't have these parameters)

As a side note on `phylum batch`: The `phylum batch` command is rarely used... But I think there might still be some scripts out there that rely on it. We could consider rewriting `phylum batch` as an extension in `community-extensions` and removing it from the core CLI.